### PR TITLE
Wrote user class

### PR DESCRIPTION
**Pull request by justin-obrien-smoothstack**

### CODE REVIEW

**Review by trevorhuis-smoothstack — APPROVED** [review]
Not much wiggle room in these entities, looks good!

File: src/main/java/com/ss/training/utopia/counter/entity/User.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+public class User implements Serializable {
+
+	private static final long serialVersionUID = -1093118640730467189L;
+
+	@Id
+	@Column
+	private int userId;
+
+	@Column
+	private String name, username, password, role;
```
Comment (trevorhuis-smoothstack) [inline src/main/java/com/ss/training/utopia/counter/entity/User.java]: I explicitly name the columns for each variable, but this looks like it isn't necessary. I think I developed that practice from the not wanting the same name in the POJO that was in the library database.

**Review by trevorhuis-smoothstack — APPROVED** [review]
Already reviewed.

**Comment by justin-obrien-smoothstack** [comment]
I made some changes since then. They're pretty unremarkable though.

**Pull request merged**